text/0033-private-network-guide.md

```diff
@@ -7,19 +7,17 @@
 
 # Summary
 
-This guide will show you how to run a local private Hathor network with its own full nodes, miner, wallet and explorer. Note that a private network will be completely separated from Hathor's mainnet and testnet.
+This guide will show you how to run a local private Hathor network with its own full nodes, tx mining service, wallet and explorer. Note that a private network will be completely separated from Hathor's mainnet and testnet.
 
 By the end of the guide, you will have a fully working private network with the following setup:
 
 - Three hathor-core full nodes connected between each other
-- One cpu-miner mining blocks in the network
-- One tx-mining-service to be used by Wallets to mine transactions
+- One tx-mining-service to be used by wallets to mine transactions and used by the full nodes to mine blocks
```

Comment:
```suggestion
- One tx-mining-service to be used by Wallets to mine transactions
- One cpu-miner connected to the tx-mining-service to mine blocks
```

What do you think?

Reply:
OK: change made:
I did not directly commit your suggestion, but added it in my commit.